### ISSUE TITLE
Allow HttpClient customization in REGAPIC

### ISSUE THREAD

**Issue by jskeet** [googleapis/gax-dotnet#465]
REGAPIC doesn't currently have any direct support for custom HttpClient or HttpClientFactory. We probably need to work out how to fix that...

**Comment by jskeet** [comment]
@amanda-tarafa Maybe we should close this, after over 3 years with no customers requesting it?

**Comment by amanda-tarafa** [comment]
Yes, closing just now. We can always reconsider if there's demand for it.
